### DISCUSSION TITLE
How to use alpha matting with server?

### DISCUSSION THREAD

**Discussion by longnt189** [danielgatis/rembg#127] [Q&A]
I will use `rembg-server` to run the server, but how can I configure alpha matting? Please help me, thank you so much.

**Comment by danielgatis** [comment]
https://github.com/danielgatis/rembg#advance-usage

**Reply by longnt189** [reply]
But it doesn't work with `embg-server`

**Reply by danielgatis** [reply]
https://github.com/danielgatis/rembg/blob/master/src/rembg/cmd/server.py#L36

```
http://localhost:5000?a=1&af=240&ab=10url=http://image.png
```

**Reply by longnt189** [reply]
Thank you very much.
